app/org/sagebionetworks/bridge/play/controllers/ConsentController.java

```diff
@@ -176,7 +170,7 @@ private Result giveConsentForVersion(int version, SubpopulationGuid subpopGuid)
         
```

Comment:
when giving consent, SharingScope gets set in line 174 above, consentService.consentToResearch